CONTRIBUTING.rst

```diff
@@ -75,15 +75,40 @@ Check the commits you have on your branch and then push them to your forked repo
         git log -10 --pretty=oneline --decorate
         git push origin your-branch-name
 
-**Step 8**: then create a pull request either from your fork, or from the official github repository. There is a pull request template that you need to edit/update before confirming the pull request creation.
-If you're proposing a **patch** that needs to be backported to a specific branch, please make sure to mention it in your pull request, such that the project responsible can properly label it. If your pull request requires further effort, please use the labels "Do not merge yet" and "Work in progress".
-The pull request title has to be meaningful as well, even though it's not used for the release notes. You might want to describe your changes and the reason behind that, it's quite helpful when we need to check a module's history.
+**Step 8**: then create a pull request either from your fork, or from the official github repository. There is a pull request template that you need to edit/update before actually creating your pull request:
+ * please make sure to provide a meaningful and short PR title;
+ * also provide a short description of the changes provided in the PR (if useful, also provide a reason for that decision and implementation)
+ * provide any possible dependency and/or external changes that are required to go with your pull request (kubernetes, COMP packages, deployments, etc).
+If your pull request requires further effort and/or it's still a work in progress, please use one or both labels like: "Do not merge yet" and "Work in progress". If you're proposing a **patch** that needs to be backported to a specific branch and/or pushed to production right away, then please make sure to mention it in your pull request such that the proper actions and labelling are done.
```

Comment:
add empty line above to separate list items from the rest of the text.

Reply:
Done. Marking as resolved as well. Please unresolve it in case I missed anything.